Calling/Controllers/CallRecordingController.cs

```diff
@@ -23,8 +24,9 @@ public class CallRecordingController : Controller
         private const string CallRecodingActiveErrorCode = "8553";
         private const string CallRecodingActiveError = "Recording is already in progress, one recording can be active at one time.";
         public ILogger<CallRecordingController> Logger { get; }
-        static Dictionary<string, RecordingInfo> recordingData = new Dictionary<string, RecordingInfo>();
+        static Dictionary<string, RecInfo> recordingData = new Dictionary<string, RecInfo>();
         private readonly string isRecordingEnabled;
+        public static string recFileFormat;
 
         public CallRecordingController(IConfiguration configuration, ILogger<CallRecordingController> logger)
```

Comment:
I would imagine that the Controller class should be fairly small just our different REST functions and we move our in memory storage, and any other references injected into the constructor

Reply:
Moved non-api related code into new classes